apps/posts/src/views/comments/components/comment-reports-modal.tsx

```
@@ -0,0 +1,109 @@
+                                </span>
+                                <LucideIcon.Dot className="shrink-0 text-muted-foreground/50" size={16} />
+                                <span className="shrink-0 text-muted-foreground">
+                                    {comment.created_at && formatTimestamp(comment.created_at)}
+                                </span>
+                                <span className="shrink-0 text-muted-foreground">on</span>
+                                <span className="min-w-0 truncate font-medium text-gray-800 dark:text-gray-400">
+                                    {comment.post?.title || 'Unknown post'}
+                                </span>
+                            </div>
+                            <div
+                                dangerouslySetInnerHTML={{__html: comment.html || ''}}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)